backend/benefit/applications/services/applications_csv_report.py

```diff
@@ -45,6 +45,20 @@ def format_bool(value):
         raise ValueError(f"Invalid value: {value}")
 
 
+def get_current_ahjo_row_field(field_name):
```

Comment:
The function name is misleading as it does not get the current ahjo row field but a getter function to get the current ahjo row field. How about `current_ahjo_row_field_getter(field_name)`, `ahjo_row_field_getter(field_name)`  or similar?